integrations/access/opsgenie/bot.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package opsgenie
+
+import (
+	"context"
+	"net/url"
+	"time"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/integrations/access/common"
+	pd "github.com/gravitational/teleport/integrations/lib/plugindata"
+)
+
+// Bot is a opsgenie client that works with AccessRequest.
+// It's responsible for formatting and Opsgenie alerts when an
+// action occurs with an access request: a new request popped up, or a
+// request is processed/updated.
+type Bot struct {
+	client      *Client
+	clusterName string
+	webProxyURL *url.URL
+}
+
+// CheckHealth checks if the bot can connect to its messaging service
+func (b *Bot) CheckHealth(ctx context.Context) error {
```

Comment:
Is it supposed to skip the test?

Reply:
➕ this raises eyebrows - does not explain why it is not supported for opsgenie. Is there no "ping" endpoint we could call that would just check that we're authenticated correctly / API is up?